self-tests.yml

```diff
@@ -540,9 +540,6 @@ tasks:
   - <<: *run-go-test-suite-with-mongodb
     tags: ["db", "race"]
     name: race-db
-  - <<: *run-go-test-suite-with-mongodb
-    tags: ["db", "race"]
-    name: race-cloud
```

Comment:
can you just put skips in for the tests that are a problem?

Reply:
Can I skip them *just* for the race variant? I'd still like them to run in the `test-cloud` case.